### ISSUE TITLE
Considering audit of p256 curve

### ISSUE THREAD

**Issue by SilentCicero** [RustCrypto/elliptic-curves#909]
We are considering an audit of the p256 curve:
https://github.com/RustCrypto/elliptic-curves/tree/master/p256

Has any other team considered auditing this library?

**Comment by tarcieri** [comment]
I don't think anyone has an audit planned for `p256` specifically, but there's an audit underway for `k256` and some of its dependencies

**Comment by xoloki** [comment]
Did you end up doing an audit of `p256` @SilentCicero ?

**Comment by tarcieri** [comment]
`p256` was audited recently. The README.md should probably be updated to reflect it:

https://reports.zksecurity.xyz/reports/near-p256/

**Comment by xoloki** [comment]
> `p256` was audited recently. The README.md should probably be updated to reflect it:

Yeah I found that once I started looking.  Very clean audit report!
